test/rules/avoid_function_literals_in_foreach_calls.dart

```diff
@@ -17,4 +21,17 @@ void main() {
   people.forEach((person) => print(person)); // LINT
 
   people.forEach(print); // OK
+
+  people
+      .where((person) => person != null)
+      .map((person) => '$person!')
+      .forEach((person) => print(person)); // OK
```

Comment:
The example may be confusing because it hits another lint (https://dart-lang.github.io/linter/lints/unnecessary_lambdas.html), and should be written as just `.forEach(print)`.
(It is literally the example given as bad for `unnecessary_lambdas`).

It's always hard to make a small meaningless example without actually being meaningless :)

Reply:
Good call, I have updated the example slightly to avoid the `unnecessary_lambdas` lint thanks!